soccer/gameplay/skills/move_direct.py

```diff
@@ -42,6 +43,12 @@ def threshold(self):
     def threshold(self, value):
         self._threshold = value
 
+    def velocity_check(self):
+        if (self.check_velocity):
+            return self.robot.vel.mag() < .05
```

Comment:
This `0.5` is in a ton of places, can we make it  a constant instead (maybe in constants.py) so we can tune it from one place?